O21.Game/Localization/Help.fs

```diff
@@ -0,0 +1,30 @@
+module O21.Localization.Help
+
+open O21.Game.Documents
+open System.Collections.Generic
+open System.Threading.Tasks
+open System.IO
+open System.Threading
+
+type HelpRequest =
+    | EnglishHelp of CancellationToken
+    | RussianHelp of (unit -> Task<DocumentFragment array>)
+
+let private readMarkdown(file: string, cancellationToken: CancellationToken) =
+    task {
+        use sr = new StreamReader (file)
+        let fragments = List<DocumentFragment> 0
+        while not sr.EndOfStream do
+            let! line = sr.ReadLineAsync cancellationToken
+            fragments.Add (match line with
+                            | "---" -> (DocumentFragment.Text(Style.Normal, "____________________________________________"))
+                            | str when (str.Length > 0 && str[0] = '#') -> DocumentFragment.Text(Style.Bold, str.Substring(1))
+                            | str -> (DocumentFragment.Text(Style.Normal, str)))
+            fragments.Add (DocumentFragment.NewParagraph)
```

Comment:
I should also note that, before merging this, we'd need some way to break the text into several lines (to avoid the overflow issue you mentioned). Let's either invent an automatic algorithm (albeit a simple one), or maybe just do that manually in the file itself.

Reply:
Edited original markdown for now